mrbelvedereci/plan/tasks.py

```diff
@@ -30,3 +33,28 @@ def run_scheduled_daily():
 @django_rq.job('short')
 def run_scheduled_hourly():
     return run_scheduled('hourly')
+
+@django_rq.job('short')
+def create_github_webhook(pk):
+    """Create a webhook if it doesn't exist yet."""
+    plan_repo = PlanRepository.objects.get(pk=pk)
+    if plan_repo.plan.type == 'manual':
+        return
+    event = 'pull_request' if plan_repo.plan.type == 'pr' else 'push'
+    callback_url = '{}/{}'.format(settings.GITHUB_WEBHOOK_BASE_URL, event)
+    exists = False
+    gh_repo = plan_repo.repo.github_api
+    for hook in gh_repo.iter_hooks():
+        if hook.config.get('url') == callback_url and event in hook.events:
+            exists = True
+            break
+    if not exists:
+        gh_repo.create_hook(
```

Comment:
i know this isn't new code, so sorry to ask lots of questions in the review.

if this fails, what happens? the worker will throw an exception - but is the sentry context fully initialized here? will i get an email if a webhook fails to create?

Reply:
this is somewhat important for jason's hypothetical user spinning up a new site.

Reply:
You're right that it's just kind of lost in a way, at least with it as a rq job it will be put into the failed job queue. Would be good to have Sentry or something send a notification, but I think that's outside the scope of this PR. I've filed https://github.com/SalesforceFoundation/mrbelvedereci/issues/148 to follow up.